mango_tree_1.js

```diff
@@ -1,32 +1,76 @@
-"use strict"
-
+'use strict'
 class MangoTree {
-  
-  // Initialize a new MangoTree
-  constructor() {
+  constructor(age,height,fruits,harvestedFruit,isHealthy) {
+    this._age = age || 0
+    this._height = height || 0.8
+    this._fruits = fruits || 0
+    this._harvestedFruit = harvestedFruit || 0
+    this._isHealthy = isHealthy || false
   }
-
   // Get current states here
+  getAge(){
+    return this._age += 1;
```

Comment:
kyaknya this._age += 1 deh pak , tu buat increment agenya nya sebanyak 1 dan akan diincrement trus sampe ketika healthnya false yaitu kondisi ketika age udah nyampe 20